state.go

```diff
@@ -913,7 +1052,34 @@ func (s *State) OnInterface(se *Session, i interface{}) (err error) {
 		if s.TrackChannels {
 			err = s.ChannelRemove(t.Channel)
 		}
-	case *MessageCreate:
+	case *ThreadCreate:
+		if s.TrackThreads {
+			msglog(LogDebug, 1, "thread created %s (%q) in %s", t.ID, t.Name, t.ParentID)
```

Comment:
We don't do this for any other objects in the state; I'd recommend dropping these logs.

Reply:
Oh sorry, was debugging state, forgot to remove.